schemas/telemetry/usage/browser-features-schema.json

```diff
@@ -14,6 +14,16 @@
           "const": "start-session-replay-recording"
         }
       }
+    },
+    {
+      "required": ["feature"],
+      "properties": {
+        "feature": {
+          "type": "string",
+          "description": "startSessionReplayRecording API",
+          "const": "force-session-replay-recording"
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** ‏since we are already monitoring the start API, what about adding the new parameter to it instead of adding a new feature?
(cf consent tracking to see how it is done)

Reply:
✔️ Adressed